### PR TITLE
feat: CS Perf Oracle artifacts description

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

**Review by madlabman — APPROVED** [review]

File: docs/staking-modules/csm/rewards.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Also, any excess [bond](./join-csm#bond) will be treated as a reward.
...
+The Performance Oracle uses the successful attestation rate `successfulAttestations / totalAssignedAttestations` as a proxy for the overall performance of a validator. A performance threshold is utilized to determine the allocation of the actual Node Operator rewards. Validators with performance above the threshold are included in the allocation pool, while the rest are not. Activation and exit events are accounted for during the Node Operator's share calculation. Once the allocation pool is formed, each validator gets a staking rewards part of `totalStakingRewardsAccumulated` proportional to its lifetime within a frame.. This effectively means that all rewards acquired by the module will be allocated among well-performers. Then, validator shares are allocated to the corresponding Node Operators, and each Operator can claim rewards for all of their validators in one go.
```
Comment (TheDZhon) [inline docs/staking-modules/csm/rewards.md:30]:
```suggestion
The Performance Oracle uses the successful attestation rate `successfulAttestations / totalAssignedAttestations` as a proxy for the overall performance of a validator. A performance threshold is utilized to determine the allocation of the actual Node Operator rewards. Validators with performance above the threshold are included in the allocation pool, while the rest are not. Activation and exit events are accounted for during the Node Operator's share calculation. Once the allocation pool is formed, each validator gets a staking rewards part of `totalStakingRewardsAccumulated` proportional to its lifetime within a frame. This effectively means that all rewards acquired by the module will be allocated among well-performers. Then, validator shares are allocated to the corresponding Node Operators, and each Operator can claim rewards for all of their validators in one go.
```

(extra period)

File: docs/staking-modules/csm/rewards.md [hunk 2]
```diff
@@ -41,6 +41,23 @@ The `frame` for the Performance Oracle report is set to 28 days. This makes the
...
+
+The Merkle tree dump can be used to construct a valid proof for Node Operators to claim their acquired rewards. For pre-generated proofs, see the [csm-rewards](https://github.com/lidofinance/csm-rewards) GitHub repository. This repository also provides detailed instructions on how to generate proof and claim rewards manually via Etherscan.
+
+A frame performance assessment log aims to achieve more transparency on rewards distribution made by Oracles. It's another JSON object that stores, among other things:
+
+- The performance threshold for a given frame;
+- The total amount of shares distributable in the frame;
+- Attestation rates of validators as "assigned" and "included" pairs;
+- The amount of shares distributed every operator in the frame.
```
Comment (TheDZhon) [inline docs/staking-modules/csm/rewards.md:55]:
```suggestion
- The amount of shares distributed to every operator in the frame.
```

File: docs/staking-modules/csm/rewards.md [hunk 3]
```diff
@@ -41,6 +41,23 @@ The `frame` for the Performance Oracle report is set to 28 days. This makes the
...
+
+There's also additional data in the log; for a full definition look at the following [typescript gist](https://gist.github.com/madlabman/33bc63843b633aa114173d7898e5fcce).
```
Comment (TheDZhon) [inline docs/staking-modules/csm/rewards.md:57]: not sure if it's a good idea to refer to the personal gist :new_moon_with_face:

File: docs/staking-modules/csm/rewards.md [hunk 4]
```diff
@@ -41,6 +41,23 @@ The `frame` for the Performance Oracle report is set to 28 days. This makes the
...
+
+One can inspect the file to ensure all the operators' `distributed` amounts are correct; for example, by using this [python gist](https://gist.github.com/madlabman/ce47a5311f004985341c6fcad53dcd0e). Interested persons can also check the attestations summaries for each validator in the log and report any discrepancies using official Lido Discord.
```
Comment (TheDZhon) [inline docs/staking-modules/csm/rewards.md:59]: and here

**Review by TheDZhon — APPROVED** [review]
GG

Lucky me if you consider a few suggestions favorably :+1:

File: docs/staking-modules/csm/rewards.md (hunk 3 above)
Comment (dgusakov) [reply docs/staking-modules/csm/rewards.md]: The gist has been moved to CSM repo

File: docs/staking-modules/csm/rewards.md (hunk 4 above)
Comment (dgusakov) [reply docs/staking-modules/csm/rewards.md]: The gist has been moved to CSM repo

**Pull request merged**